### PR TITLE
Add new camera models (and the concept of an "Outdoor Camera")

### PR DESCRIPTION
**Pull request by Abductist**
Added in the model type of the 1080p version of the SimpliCam

**Describe what the PR does:**

**Does this fix a specific issue?**

Fixes https://github.com/bachya/python-simplisafe/issues/<ISSUE ID>
  
**Checklist:**

- [ ] Confirm that one or more new tests are written for the new functionality.
- [x] Run tests and ensure everything passes (with 100% test coverage).
- [ ] Update `README.md` and `docs/` with any new documentation.
- [x] Add yourself to `AUTHORS.md`.

### CODE REVIEW

**Review by bachya — DISMISSED** [review]
~~Thank you!~~

File: simplipy/camera.py [hunk 1]
```diff
@@ -12,12 +12,14 @@
 DEFAULT_AUDIO_ENCODING = "AAC"
 
 CAMERA_MODEL_CAMERA = "camera"
+CAMERA_MODEL_CAMERA_1080 = "camera"
```
Comment (bachya) [inline simplipy/camera.py:15]: We shouldn't have constants with the same value. Is there any reason we can't use the existing `CAMERA_MODEL_CAMERA`?

**Review by bachya — CHANGES_REQUESTED** [review]

File: simplipy/camera.py (hunk 1 above)
Comment (Abductist) [reply simplipy/camera.py]: No, that's a fair point. I'll remove that.

**Review by Abductist — COMMENTED** [review]
Added the new outdoor camera's model type to this request

**Review by bachya — APPROVED** [review]
Thank you!

**Pull request merged**